### PR TITLE
PR: Move CI to github actions on branch master

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

**Comment by goanpeca** [comment]
Need to think how to collect all coverage into a single batch, but that is work for another PR

**Comment by ccordoba12** [comment]
> Need to think how to collect all coverage into a single batch, but that is work for another PR

Right, no problem about it. Let's merge this one.

File: .github/workflows/linux-tests.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false 
       matrix:
-        PYTHON_VERSION: ['2.7', '3.6', '3.7', '3.8']
```
Comment (ccordoba12) [inline .github/workflows/linux-tests.yml]: I'd like to maintain compatibility with Python 2 for a little longer here so people can still evaluate their Python 2 code. Since that doesn't represent a big effort for this small package, please revert these changes.

Comment (goanpeca) [reply .github/workflows/linux-tests.yml]: Done

File: setup.py [hunk 2]
```diff
@@ -38,7 +38,6 @@ def get_version(module='spyder_kernels'):
 REQUIREMENTS = [
     'cloudpickle',
     'ipykernel',
-    'ipykernel>=5.1.3; python_version>"2"',
```
Comment (goanpeca) [inline setup.py]: @ccordoba12 what should I leave here?

Comment (ccordoba12) [reply setup.py]: The same, and please also revert the classifier deletions you did below.

Comment (goanpeca) [reply setup.py]: Ok
